gui/src/components/Taipy/tableUtils.tsx

```diff
@@ -197,7 +197,7 @@ const getCellProps = (col: ColumnDesc, base: Partial<TableCellProps> = {}): Part
     switch (col.type) {
         case "int":
         case "float":
-            base.align = "right";
```

Comment:
@JaufreyTpy Here you change numeric value alignment for all columns with or without styleKit.
If it's really what's wanted, I guess we should remove the int and floats case (L 198->201)

Reply:
Yeah, that's wanted, but I wasn't sure of what I was doing there, so I left it for the moment.
I'll remove it all right, because I can't see a reason why we would want to mess with alignment on numbers like before. The more normalized it is, the easiest it is to interact with it.

Btw, is the
```
case "bool":
            base.align = "center";
            break;
```
for centering a toggle button (switch) or something like that ? I don't know how to test that case.

Reply:
create a column of boolean values